src/ui/common/src/components/operators/operatorParametersOverview.tsx

```diff
@@ -28,11 +28,20 @@ const OperatorParametersOverview: React.FC<Props> = ({
 
     // These checks tries to distinguish googlesheet vs relational
     // extracts based on the fields of type union exParams.
-    if ('query' in exParams) {
+    if ('query' in exParams || 'queries' in exParams) {
+      const relationalParams = exParams as RelationalDBExtractParams;
+      if (!!relationalParams.queries) {
+        return (
+          <Typography variant="body2" color={textColor}>
+            {`${relationalParams.queries.length} chained queries.`}
```

Comment:
It'll be more informative if we can find a way to nicely show the content on all chained queries.